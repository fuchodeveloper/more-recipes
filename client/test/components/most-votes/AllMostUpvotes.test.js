import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import { AllMostUpvotes }
  from '../../../components/most-upvotes/AllMostUpvotes';

let props;

const setup = () => {
  props = {
    details: {
      upVotes: 0,
      image: 'https://res.cloudinary.com/fuchodeveloper/image/upload/'
    + 'v1516760699/noodles_c6ltkq.jpg',
      name: 'jollof',
      direction: 'rice, beans',
      views: 0,
      id: 1
    }
  };
  return shallow(<AllMostUpvotes {...props} />);
};

describe('MostUpvotesPage component', () => {
  describe('MostUpvotesPage component snapshot', () => {
    it('should render with right amount of elements', () => {
      const { wrapper } = setup();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
