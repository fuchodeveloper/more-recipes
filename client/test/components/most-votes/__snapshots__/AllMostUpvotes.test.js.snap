// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MostUpvotesPage component MostUpvotesPage component snapshot should render with right amount of elements 1`] = `undefined`;
